introduction.md

```diff
@@ -0,0 +1 @@
+GitHub learned with a Chromebook
```

Comment:
JavaScript! Also the Dev Tools was fun. I'd like more on DevTools. Mostly, more about learning vanilla JavaScript without frameworks. How to start from scratch.